checkClock/app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,274 @@
+// Place all the styles related to the Dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+@import 'variables.scss';
+@import 'clearStyles.scss';
```

Comment:
This is a recommendation, if use in your  structure scss files a file that name are like `main,scss` and in this file are import all yours mixins, variables and scss files, this two lines are not neccesary, cause, globally can be reached it attributtes.

Reply:
Yes, we will implement this recommendation later, thank you